### DISCUSSION TITLE
中国是民主国家

### DISCUSSION THREAD

**Discussion by Jesse3692** [WeNeedHome/SummaryOfLoanSuspension#349] [General]
中国是一个民主国家，西方的都是假民主

**Comment by lucas-acc** [comment]
你说民笑.jpg

**Comment by GitHeP** [comment]
哈哈哈哈哈哈哈哈

**Comment by Accelerator6666** [comment]
你说的东西我想都不敢想，太刺激了

**Comment by oceanzhang860411** [comment]
偷着乐系列

**Comment by C-Joey** [comment]
全过程民主

**Comment by zhw173** [comment]
不加个狗头？

**Reply by Baz2013** [reply]
核酸大国

**Comment by LixHu** [comment]
民煮国家

**Comment by mehhss** [comment]
确实

**Comment by Kamijoucen** [comment]
哈哈哈哈哈哈哈哈

**Comment by Joker-smile** [comment]
我都看笑了

**Comment by wd30x** [comment]
没有，没有，没有，通过！

**Reply by xijinping0** [reply]
补充一个视频（点击图片播放）：

[![没有通过](https://img.youtube.com/vi/Fdc-m5hTRpk/0.jpg)](https://youtu.be/Fdc-m5hTRpk?t=12)

https://github.com/xijinping0

**Comment by lettermessenger** [comment]
阿Q

**Comment by 505Games-oss** [comment]
闹得欢，拉清单

**Reply by xijinping0** [reply]
补充一个视频（点击图片播放）：

[![拉清单](https://img.youtube.com/vi/iBg0UhhV72E/0.jpg)](https://youtu.be/iBg0UhhV72E?t=113)

https://github.com/xijinping0

**Comment by xijinping0** [comment]
假民主还要对我们指手画脚，吃饱了没事干（点击图片播放）：

[![外国人吃饱了没事干](https://img.youtube.com/vi/aG903lJtC7M/0.jpg)](https://youtu.be/aG903lJtC7M?t=40)
